### ISSUE TITLE
Template property cannot be empty

### ISSUE THREAD

**Issue by tomasherceg** [riganti/dotvvm#1617]
If I have a control with an `ITemplate` property which is marked with `[MarkupOptions(Required=true)`, the usage validation fails even when the element is present on the page and is empty. I think we should allow empty templates - if they are explicitly specified, they don't violate the required requirement for the property.

```
<bp:GridViewTemplateColumn>
    <ContentTemplate></ContentTemplate>
</bp:GridViewTemplateColumn>
```

**Comment by exyi** [comment]
#1597

**Comment by exyi** [comment]
resolved by #1597, should be in the  4.1.4 release
